backend/workflow_manager/workflow_v2/models/execution.py

```diff
@@ -48,6 +48,9 @@ class Type(models.TextChoices):
     )
     # TODO: Restrict with an enum
     status = models.CharField(default="", db_comment="Current status of execution")
+    result_acknowledged = models.BooleanField(
```

Comment:
NIT: @muhammad-ali-e do you see a need to store the time at which the result was acknowledged? Maybe that alone can indicate whether the RESULT was obtained while also storing an extra info

Reply:
Good suggestion. @muhammad-ali-e right now we might not have a use. But may be in future this could come in handy. But due to current priority my suggestion would be to keep it as another ticket and we can revisit it once the high priority items are over.

Reply:
I also thought about this during development. We should do this later.